wiki/conf/crontab.ugly

```diff
@@ -6,7 +6,7 @@ PATH=/usr/local/bin:/usr/bin:/bin
 PYTHON=/data/vhost/!!(*= $vhost *)!!/virtualenv-wiki/bin/python
 
 24 5 * * * !!(*= $user *)!! $PYTHON /data/vhost/!!(*= $vhost *)!!/internal/wiki/bin/wikibot-fixup.py
-36 3 * * * !!(*= $user *)!! $PYTHON /data/vhost/!!(*= $vhost *)!!/internal/wiki/bin/wikibot-pipedrive.py
+36 3 * * * !!(*= $user *)!! export PYTHONIOENCODING=utf-8; $PYTHON /data/vhost/!!(*= $vhost *)!!/internal/wiki/bin/wikibot-pipedrive.py
```

Comment:
Better solution would be to use python3 instead ;-)

Reply:
No doubt you are right, but I think I'd like to get this change in for now so that it runs - there are a couple of issues covering more detailed improvements (e.g. #38 and #39), so perhaps we can revisit this.

Reply:
Maybe I'll have a chat with @jacksonj04 about revisiting #40.